README.md

```diff
@@ -151,6 +151,13 @@ require("typescript-tools").setup {
     -- by default code lenses are displayed on all referencable values and for some of you it can
     -- be too much this option reduce count of them by removing member references from lenses
     disable_member_code_lens = true,
+    -- JSXCloseTag
+    -- WARNING: it is disabled by default (maybe you configuration or distro already uses nvim-auto-tag,
+    -- that maybe have a conflict if enable this feature. )
```

Comment:
Add this to [README](https://github.com/pmizio/typescript-tools.nvim/blob/master/README.md?plain=1#L156) also.